utils/notification.ts

```diff
@@ -142,6 +142,37 @@ const successNH = (): NHResultSuccess =>
     kind: "SUCCESS"
   });
 
+const dictionaryIsEmpty = (
+  obj: Azure.ServiceBus.Dictionary<string | unknown>
+): boolean => Object.keys(obj).length === 0;
+
+const handleResponseOrError = (
+  resolve: (
+    value:
+      | { readonly kind: "SUCCESS" }
+      | PromiseLike<{ readonly kind: "SUCCESS" }>
+  ) => void,
+  reject: (reason?: unknown) => void
+) => (err: Error | null, response: Azure.ServiceBus.Response): void =>
+  err == null
+    ? resolve(successNH())
+    : reject(
+        `[error message: ${err.message}][response: ${response?.statusCode ??
+          ""} - ${
+          !response?.body || dictionaryIsEmpty(response?.body)
+            ? "No response body"
+            : response?.body
+        }]`
```

Comment:
Cose style: try to avoid complex nesting in string templates; instead, compose intermediate variables.

Reply:
I tried to avoid this by extracting a new `composeNHErrorMessage` function. Let me know